app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -131,7 +131,7 @@ def connect
       :rev      => '6.5',
     }
 
-    require 'rbvmomi/vim'
+    require 'rbvmomi'
```

Comment:
The base rbvmomi module requires rbvmomi/connection, rbvmomi/sso, and rbvmomi/vim (aka everything needed for connecting) but not pbm and sms (aka other endpoints that we don't want to pull in) which was the point of only requiring rbvmomi/vim here. Now rbvmomi/vim depends on rbvmomi/connection so we should really be requring the base